### PR TITLE
Change password sidletskyi

### PR DESCRIPTION
**Pull request by SidletskyiOlexandr**

### CODE REVIEW

**Review by vladtymo — APPROVED** [review]

File: src/app/core/services/authentication.service.ts [hunk 1]
```diff
@@ -158,4 +164,13 @@ export class AuthenticationService {
       this.currentUser = new UserInfo();
     }));
   }
+
+  sendResetPasswordToken(forgotPassword: ForgotPassword): Observable<void>{
+    return this.http.get<void>(this.forgotPasswordUrl + forgotPassword.email);
+  }
+
+  resetPassword(resetPassword: ResetPassword): Observable<void>{
+    return this.http.post<void>(this.resetPasswordUrl, resetPassword);
```
Comment (pashunskyi) [inline src/app/core/services/authentication.service.ts]: Are you sure we need to use the POST method?

File: src/app/configs/api-endpoints.ts [hunk 2]
```diff
@@ -8,6 +8,8 @@ export const loginUrl = authenticationServiceUrl + 'login';
 export const refreshTokenUrl = authenticationServiceUrl + 'refresh-token';
 export const logoutUrl = authenticationServiceUrl + 'logout';
 export const twoStepVerificationUrl = authenticationServiceUrl + 'login-two-step';
+export const forgotPasswordUrl = authenticationServiceUrl + 'forgot-password/';
+export const resetPasswordUrl = authenticationServiceUrl + 'reset-password';
```
Comment (pashunskyi) [inline src/app/configs/api-endpoints.ts]: Are you sure we can use verbs in links?

File: src/app/core/services/authentication.service.ts (hunk 1 above)
Comment (SidletskyiOlexandr) [reply src/app/core/services/authentication.service.ts]: On server side this method is POST. Do I need change it on server side?

Comment (pashunskyi) [reply src/app/core/services/authentication.service.ts]: Yes, I think it must be PUT method

**Review by pashunskyi — APPROVED** [review]

**Pull request merged**